.rubocop.yml

```diff
@@ -0,0 +1,666 @@
+
+Style/ModuleFunction:
+  Enabled: false
+
+Lint/Debugger:
+  Enabled: false
+
+Style/IfWithSemicolon:
+  Enabled: false
+
+Style/Encoding:
+  Enabled: false
+
+Metrics/PerceivedComplexity:
+  Enabled: false
+
+Style/SymbolProc:
+  Enabled: false
+
+Style/SpaceInsideRangeLiteral:
+  Enabled: false
+
+Style/InfiniteLoop:
+  Enabled: false
+
+Style/BarePercentLiterals:
+  Enabled: false
+
+Style/PercentQLiterals:
+  Enabled: false
+
+Style/MultilineBlockLayout:
+  Enabled: false
+
+Metrics/AbcSize:
+  Enabled: false
+
+Style/MutableConstant:
+  Enabled: false
+
+Style/BlockDelimiters:
+  Enabled: false
+
+Style/EmptyLinesAroundClassBody:
+  Enabled: false
+
+Style/ConditionalAssignment:
+  Enabled: false
+
+Style/ExtraSpacing:
+  Enabled: false
+
+Style/EmptyLinesAroundBlockBody:
+  Enabled: false
+
+Style/EmptyLinesAroundModuleBody:
+  Enabled: false
+
+Style/MultilineOperationIndentation:
+  Enabled: false
+
+Style/EmptyElse:
+  Enabled: false
+
+Style/StringLiteralsInInterpolation:
+  Enabled: false
+
+Style/MultilineMethodCallIndentation:
+  Enabled: false
+
+Metrics/ModuleLength:
+  Enabled: false
+
+Style/EmptyLinesAroundMethodBody:
+  Enabled: false
+
+Lint/IneffectiveAccessModifier:
+  Enabled: false
+
+Performance/StringReplacement:
+  Enabled: false
+
+Style/ClosingParenthesisIndentation:
+  Enabled: false
+
+Style/UnneededInterpolation:
+  Enabled: false
+
+Style/ElseAlignment:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/FirstParameterIndentation:
+  Enabled: false
+
+Style/IfInsideElse:
+  Enabled: false
+
+Style/IndentAssignment:
+  Enabled: false
+
+Style/SpaceAroundBlockParameters:
+  Enabled: false
+
+Style/ParallelAssignment:
+  Enabled: false
+
+Performance/RedundantBlockCall:
+  Enabled: false
+
+Style/IdenticalConditionalBranches:
+  Enabled: false
+
+Performance/RedundantMatch:
+  Enabled: false
+
+Style/CommandLiteral:
+  Enabled: false
+
+Performance/Casecmp:
+  Enabled: false
+
+Lint/NestedMethodDefinition:
+  Enabled: false
+
+Style/SpaceInsideStringInterpolation:
+  Enabled: false
+
+Performance/RedundantMerge:
+  Enabled: false
+
+Performance/ReverseEach:
+  Enabled: false
+
+Style/NestedModifier:
+  Enabled: false
+
+Lint/NonLocalExitFromIterator:
+  Enabled: false
+
+Performance/Count:
+  Enabled: false
+
+Style/NestedParenthesizedCalls:
+  Enabled: false
+
+Style/RescueEnsureAlignment:
+  Enabled: false
+
+Lint/DuplicateMethods:
+  Enabled: false
+
+Performance/RangeInclude:
+  Enabled: false
+
+Style/TrailingUnderscoreVariable:
+  Enabled: false
+
+Lint/LiteralInInterpolation:
+  Enabled: false
+
+Performance/DoubleStartEndWith:
+  Enabled: false
+
+Performance/RedundantSortBy:
+  Enabled: false
+
+Performance/TimesMap:
+  Enabled: false
+
+Style/InitialIndentation:
+  Enabled: false
+
+Style/StructInheritance:
+  Enabled: false
+
+Style/SymbolLiteral:
+  Enabled: false
+
+Style/IfUnlessModifierOfIfUnless:
+  Enabled: false
+
+Style/ZeroLengthPredicate:
+  Enabled: false
+
```

Comment:
Instead of adding a hug exception file (which it looks like was pulled from Puppet) did you know about the Rubocop Todo file?